lambdas/harvesters/ror/processor.rb

```diff
@@ -0,0 +1,215 @@
+# frozen_string_literal: true
+
+# Docs say that the LambdaLayer gems are found mounted as /opt/ruby/gems but an inspection
+# of the $LOAD_PATH shows that only /opt/ruby/lib is available. So we add what we want here
+# and indicate exactly which folders contain the *.rb files
+my_gem_path = Dir['/opt/ruby/gems/**/lib/']
+$LOAD_PATH.unshift(*my_gem_path)
+
+require 'digest'
+require 'uri'
+require 'zip'
+
+require 'aws-sdk-s3'
+
+require 'uc3-dmp-api-core'
+require 'uc3-dmp-cloudwatch'
+require 'uc3-dmp-dynamo'
+require 'uc3-dmp-external-api'
+
+module Functions
+  # A service that fetches the latest ROR file
```

Comment:
L21: Same here. If you could just add the description from your PR here, like " It fetches the latest ROR ZIP from S3, unzips it and then processes the records it is instructed to (e.g. 50,001 - 100,000). Processing means that it adds/updates each record into the new ExternalData DynamoDB Table".